tests/modules/oid.rs

```diff
@@ -14,24 +14,6 @@ fn deserialize() {
     assert_eq!(1_122_867 as u32, oid.counter());
 }
 
-#[test]
```

Comment:
This test is used to verify `ObjectId::timestamp`, which will be removed as part of RUST-316